swarm/api/storage.go

```diff
@@ -34,10 +34,10 @@ type Response struct {
 //
 // DEPRECATED: Use the HTTP API instead
```

Comment:
@zelig @lmars Why is this still here in the Network Rewrite if it's deprecated? We break a lot of other stuff, so maybe a good occasion to get rid of it?